patches/kernelci-core/staging.kernelci.org/0005-STAGING-use-staging-branch-for-buildroot-with-reduce.patch

```diff
@@ -1,18 +1,18 @@
-From bb55b58ab4fc2159a9e6c9fab71150e212f33931 Mon Sep 17 00:00:00 2001
+From 934df86dfc463fe4e8a426a898eae2292f823d31 Mon Sep 17 00:00:00 2001
 From: "kernelci.org bot" <bot@kernelci.org>
 Date: Tue, 22 Mar 2022 17:38:32 +0000
-Subject: [PATCH 5/7] STAGING use staging branch for buildroot with reduced
+Subject: [PATCH 5/6] STAGING use staging branch for buildroot with reduced
  arch list
 
 ---
  config/core/rootfs-configs.yaml | 2 +-
  1 file changed, 1 insertion(+), 1 deletion(-)
 
 diff --git a/config/core/rootfs-configs.yaml b/config/core/rootfs-configs.yaml
-index c81de2786..71f7eb33f 100644
+index f5c55986f..c10225213 100644
 --- a/config/core/rootfs-configs.yaml
 +++ b/config/core/rootfs-configs.yaml
-@@ -43,7 +43,7 @@ rootfs_configs:
+@@ -47,7 +47,7 @@ rootfs_configs:
```

Comment:
Note that there are a couple patch block header changes (as well as subject etc.) -- please double-check this is OK.